examples/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Matrix SDK Examples
```

Comment:
readme files usually have an all-uppercase filename (except the extension). For consistency I think we should do the same thing here.